specification/metadata/metadata_generator.md

```diff
@@ -0,0 +1,37 @@
+# FOCUS Metadata Generator
+The FOCUS metadata provider logical section provides metadata about the provider of the FOCUS data. 
+
+## Logical Section: Schema Generator
+
+## Schema Definition Metadata
+
+* ### billing_generator
+    * use: Human readable name of the entity that is generating the billing data 
+    * type: STRING
+* ### provider_prefix
+    * use: Shortened name of the provider supplied to be used as the prefix for tags and may be used to modify the name of custom columns.
```

Comment:
I believe the column prefix and the tag prefix might be different, so they might two different meta-data columns.
There might be cases where some providers have different prefixes for tags, so a list type might be needed (JSON array of strings here).